library/Rules/Email.php

```diff
@@ -5,6 +5,14 @@ class Email extends AbstractRule
 {
     public function validate($input)
     {
-        return is_string($input) && filter_var($input, FILTER_VALIDATE_EMAIL);
+        if (class_exists('\Egulias\EmailValidator\EmailValidator')) {
+            $strictValidator = new \Egulias\EmailValidator\EmailValidator();
+
+            $validationResult = $strictValidator->isValid($input);
+        } else {
+            $validationResult = is_string($input) && filter_var($input, FILTER_VALIDATE_EMAIL);
+        }
```

Comment:
@egulias, can you avoid to use else, you can just return `$strictValidator->isValid($input);`